### DISCUSSION TITLE
post/2020-08-08-use_loki_as_k8s_log_collector_2/

### DISCUSSION THREAD

**Discussion by giscus** [zcong1993/blog#67] [Announcements]
# post/2020-08-08-use_loki_as_k8s_log_collector_2/

上篇文章介绍了日志收集的背景和 loki 的优点, 真正部署的时候基本是一键的(开箱即用部署简单确实是优势)感觉没什么技术含量, 本章将介绍如何使用它来收集容器内日志文件.

https://blog.cong.moe/post/2020-08-08-use_loki_as_k8s_log_collector_2/

**Comment by shouji1128955** [comment]
非常棒，很少有这种高质量的文档
